client/my-sites/theme/controller.jsx

```diff
@@ -53,7 +52,7 @@ export function fetchThemeDetailsData( context, next ) {
 
 export function details( context, next ) {
 	const { slug, section } = context.params;
-	if ( startsWith( context.prevPath, '/themes' ) ) {
+	if ( context.prevPath && context.prevPath.startsWith( '/themes' ) ) {
```

Comment:
We could use optional chaining here to shorten this, no?

Reply:
This way it's safer, because [`context.prevPath` can be set to `false`](https://github.com/Automattic/wp-calypso/blob/trunk/client/boot/common.js#L67). Optional chaining wouldn't prevent '.startsWith` being called on the boolean.